modules/nf-core/plink2/vcf/meta.yml

```
@@ -3,6 +3,7 @@ description: Import variant genetic data using plink2
 keywords:
   - plink2
   - import
+  - variant genetic
```

```suggestion
  - "variant genetic"
```

Not sure this won't break things @mashehu ?

---

Why should it break things? This is a valid yaml string, no?